stable/grafana/templates/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/dashboards-config: {{ include (print $.Template.BasePath "/dashboards-configmap.yaml") . | sha256sum }}
       {{- range $key, $value := .Values.server.annotations }}
```

Comment:
If there are no visible changes in your deployment.yaml Kubernetes will not do a new release even if the depending Configmap has been changed. These two are treaded independently by Kubernetes and this is a "hack" in order for Kubernetes to realise that the Deployment needs to be re-deployed.